tests/constants/admincli/cmd.go

```diff
@@ -82,4 +85,22 @@ const (
 
 	// ReadWriteAccess read-write rights for the volume
 	ReadWriteAccess = "read-write"
+
+	// ListVmgroupVMs referring to vmdkops_admin vmgroups vm ls
+	ListVmgroupVMs = vmdkopsAdmin + "vmgroup vm ls --name="
+
+	// VMlist VM of the vmgroup
+	VMlist = " --vm-list="
+
+	// AddDatastoreToVMgroup adds datastore to vmgroup
+	AddDatastoreToVMgroup = vmdkopsAdmin + "vmgroup access add --name="
+
+	// AddDatastoreAccess adds datastore to vmgroup
+	AddDatastoreAccess = vmdkopsAdmin + "vmgroup access add --name="
+
+	// NotApplicable or NotAvailable
+	NotApplicable = "N/A"
```

Comment:
Yes, that's how I had done initially - made it local to test. And then I was asked to put it into separate constant file. Now again bring it back to local test. Gosh!!